about.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="assets/utiles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header">
+      <!-- Desktop menu -->
+      <div class="header-top">
+        <ul>
+          <li>
+            <a href=""
+              ><img
+                src="assets/icones/1.png"
+                alt="github"
+                width="30"
+                height="30"
+            /></a>
+          </li>
+          <li>
+            <a href=""
+              ><img
+                src="assets/icones/4.png"
+                alt="tweeter"
+                width="30"
+                height="30"
+            /></a>
+          </li>
+          <li><a href="">Login</a></li>
+          <li><a href="">Sign Up</a></li>
+        </ul>
+      </div>
+      <div class="desktop-menu">
+        <a href="index.html"
+          ><img src="assets/images/GE.png" alt="" width="120" class="logo"
+        /></a>
+        <nav>
+          <ul class="desktop-nav">
+            <li><a href="about.html">About</a></li>
+            <li><a href="">Program</a></li>
+            <li><a href="">Join</a></li>
+            <li><a href="">Support</a></li>
+            <li><a href="">News</a></li>
+            <li class="ge-campaigne"><a href="">GE campaign</a></li>
+          </ul>
+        </nav>
+      </div>
+      <button class="menu-button">
+        <img src="assets/icones/menu.png" alt="menu" width="45" height="45" />
+      </button>
+
+      <div class="nav-active">
+        <button class="menu-button">
+          <img
+            src="assets/icones/cancel.png"
+            alt="close menu"
+            width="40"
+            height="40"
+          />
+        </button>
+        <ul class="list-nav">
+          <li><a href="index.html" class="nav-list">Home</a></li>
+          <li><a href="about.html" class="nav-list">About</a></li>
+          <li><a href="">Program</a></li>
+          <li><a href="">Join</a></li>
+          <li><a href="">Support</a></li>
+          <li><a href="">News</a></li>
+      </ul>
+      </div>
+    </header>
+    <!-- about-section-one -->
+    <section class="about-section">
+      <div class="about-us-container">
+        <h2 class="hello-text">"Hello sharing world"</h2>
+        <h1 class="orange about-title">GOLDEN EVENT WEBSITE</h1>
```

Comment:
### Your Wave
![short-title-about-page](https://user-images.githubusercontent.com/93039359/211143490-d8ac82d0-5196-4e44-becd-aa7f4c87ba61.PNG)

### Design Wave
![short-title-about-page](https://user-images.githubusercontent.com/93039359/211143502-d8121d6f-2195-4402-9b1c-867c1ca46354.PNG)

- [ ] You can see the you are not using the correct color. Kindly use the correct follow the design 😊